iOS/Flows/Conversation/Status View/MessageReadView.swift

```diff
@@ -70,27 +70,31 @@ class MessageReadView: MessageStatusContainer {
             // text, no image
             let maxWidth = self.maxWidth - self.padding.value.doubled - self.imageView.width
             self.label.setSize(withWidth: maxWidth)
-            self.label.pin(.left, offset: self.padding)
-            self.label.centerOnY()
```

Comment:
Adjusting this so we always have a width before we try and position anything horizontally.